src/main/java/qna/domain/DeleteHistories.java

```diff
@@ -0,0 +1,33 @@
+package qna.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class DeleteHistories {
+
+    private final List<DeleteHistory> deleteHistories = new ArrayList<>();
```

Comment:
미리 빈 배열로 내부 컬렉션을 초기화하게 되면 일급 컬렉션을 불변 객체로 만들 수 없게됩니다!
생성자를 통해서 초기화를 진행해보시면 좋을 것 같아요!
생성자로 초기화해주실 때 Collections.unmodifiableList()를 활용해보시면 좋을 것 같아요
    }/

```suggestion
    private final List<DeleteHistory> deleteHistories;
```

Reply:
아 그렇네요 이미 변경 가능하니까ㅠ 생성 시점에 `Collections.unmodifiableList()`하면 되는군요. 감사합니다!